common.blocks/popup/popup.tests/gemini.blocks/test-nested/test.styl

```diff
@@ -1 +0,0 @@
-../../simple.blocks/test/test.styl
```

Comment:
@andrewblond but how is freeze work now? What are the problems to use borschik for such cases?